### PR TITLE
Created the errRouter, indexRouter and weatherRouter files.

### PR DESCRIPTION
**Pull request by hamzam1478**
- Created 3 new files within the new folder routers.
- Allowed user to get local weather typing in their city and countryCode.
- Told to change the branches around as I was trying to merge main into hamza.

### CODE REVIEW

**Comment by hamzam1478** [comment]
👍

File: hamza/routes/indexRouter.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const getWeather = require("../lib/getWeather");
+
+router.get("/", async (req, res) => {
+  let data = await getWeather("Manchester", "uk");
+  let name = data.name;
+  let description = data.weather[0].description;
+  let temp = data.main.temp;
+  let feels_like = data.main.feels_like;
+
```
Comment (nmunro) [inline hamza/routes/indexRouter.js:12]: I like that you've left a line break between declaring variables and rendering the template, it makes the code very easy to read, well done!

File: hamza/routes/weatherRouter.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const router = express.Router();
+
```
Comment (nmunro) [inline hamza/routes/weatherRouter.js:3]: I also like that you have a line break between dependencies we install and require and libraries we write and require, again it makes the code readable and logical

File: hamza/views/weather.hbs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <label for="code">country code: </label>
+    <input type="text" name="countryCode">
+    <input type="submit" value="submit">
+</form>
+
+<br>
+
+{{#if err}}
+<h2>{{err}}</h2>
+
```
Comment (nmunro) [inline hamza/views/weather.hbs:15]: I'd remove the line break here and move it below the {{/if}}

**Review by nmunro — APPROVED** [review]

**Pull request closed without merging**